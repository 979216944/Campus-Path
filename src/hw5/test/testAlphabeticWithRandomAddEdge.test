# Tests that list edges in alphabetic order

# Create graph
CreateGraph g

# Add nodes to g
AddNode g a
AddNode g b
AddNode g c
AddNode g d
AddNode g e

# Add Edges to g in random order
AddEdge g a c ac
AddEdge g a e ae
AddEdge g a a aa
AddEdge g a b ab
AddEdge g a c acc
AddEdge g a d ad
AddEdge g a e aee
AddEdge g a e aeee

# List the children of a in g
ListChildren g a