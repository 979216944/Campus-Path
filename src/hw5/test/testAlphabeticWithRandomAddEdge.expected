# Tests that list edges in alphabetic order

# Create graph
created graph g

# Add nodes to g
added node a to g
added node b to g
added node c to g
added node d to g
added node e to g

# Add Edges to g in random order
added edge ac from a to c in g
added edge ae from a to e in g
added edge aa from a to a in g
added edge ab from a to b in g
added edge acc from a to c in g
added edge ad from a to d in g
added edge aee from a to e in g
added edge aeee from a to e in g

# List the children of a in g
the children of a in g are: a(aa) b(ab) c(ac) c(acc) d(ad) e(ae) e(aee) e(aeee)